docs/contribution_guide_usability_testing.md

```diff
@@ -0,0 +1,109 @@
+
+> :x:
+>
+> Edit your profile picture to a newer one.
+
+This scenario is not well contextualized. In case the button for editing the profile picture is called "Edit", it suggests where the user should click.
+
+For this situation, a good example of a scenario could be:
+
+> :heavy_check_mark:
+>
+> You received an e-mail asking you to update your personal information because your profile picture is outdated. Please, look for this option in the application.
+
+This scenario puts a reason for doing the action on the application and does not suggest the user click on "Edit" instantly.
+
+Once you have finished designing your test, test the test: review it by running the test like you expect the users to do. This is important to check if the test flow and tasks make sense.
+
+## 2. Running the test sessions
+
+### Users
+
+Find some users accordingly to the requirements. Usually, 5 users are enough for a test iteration, but you can contribute with as many users as you can or want to. Collect some data from the users, like their familiarity with similar software, or their technical background: it can be useful later to evaluate results.
+
+### Preparation
+
+Set the environment up like described on the test requirements. Run the test like it is expected from the users, so that you'll know what you have to pay attention to.
+
+Start the session by providing context to the user on what the software is about and what the tests are for. It's a good idea to ask the user permission to record the session so that you can focus on observing the task execution and rewatch the session later or show the videos to the software designers.
+
+Encourage the user to think aloud, telling what they are looking for, what they are trying to do, to help you follow their rationale. It is common that sometimes they forget to think aloud, so you can ask questions about what they are doing to remind them.
+
+Explain how the session is going to be, and make it clear for the user that what is being tested is the software, not their knowledge or technology skills, so if they fail, it's the interface fault. Allow them to give up on a task if they are stuck and can't figure it out.
+
+Explicitly tell the user that you expect them to be honest and that you won't be offended if the interface is not good.
+
+### Test execution
+
+For each scenario, read it aloud to the user and watch they interact with the interface. Don't give hints, let the user try on their own. If they ask questions, guide them with other questions, be careful not to bias the test result by helping the user.
+
+After each scenario, you can ask some follow-up questions to gather some information that may come up only when the task is finished, like "where were you expecting to find that button first?", "which icon were you looking for?".
+
+## 3. Reporting results
+
+Add your results to the corresponding issue comments. Once all the tests are finished, add the conclusions to the issue template.
```

Comment:
Ditto about the placement of the results.

Reply:
I like this!